src/playground/Widgets/TasksList/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Circle, InProgressTask } from "@/icons"
+import { useMemo } from "react"
+
+type TaskStatus = "in-progress" | "due" | "no-due"
+
+type TaskItemProps = {
+  title: string
+  status: TaskStatus
+}
+
+function TaskItem({ title, status }: TaskItemProps) {
+  return (
+    <div className="flex flex-row items-start gap-2">
+      {(status === "due" || status === "no-due") && (
+        <Circle width={24} opacity={status === "no-due" ? 0.4 : 1} />
```

Comment:
Why the opacity? Are you using it to change the color? We have shades of grey you can use.